data/Hints/tournament.json

```diff
@@ -10,18 +10,37 @@
     "dungeons_barren_limit": 1,
     "named_items_required":  true,
     "distribution":          {
-        "trial":      {"order":  1, "weight": 0.0, "fixed":   0, "copies": 2},
-        "always":     {"order":  2, "weight": 0.0, "fixed":   0, "copies": 2},
-        "woth":       {"order":  3, "weight": 0.0, "fixed":   5, "copies": 2},
-        "barren":     {"order":  4, "weight": 0.0, "fixed":   3, "copies": 2},
-        "entrance":   {"order":  5, "weight": 0.0, "fixed":   4, "copies": 2},
-        "sometimes":  {"order":  6, "weight": 0.0, "fixed": 100, "copies": 2},
-        "random":     {"order":  7, "weight": 9.0, "fixed":   0, "copies": 2},
-        "item":       {"order":  0, "weight": 0.0, "fixed":   0, "copies": 2},
-        "song":       {"order":  0, "weight": 0.0, "fixed":   0, "copies": 2},
-        "overworld":  {"order":  0, "weight": 0.0, "fixed":   0, "copies": 2},
-        "dungeon":    {"order":  0, "weight": 0.0, "fixed":   0, "copies": 2},
-        "junk":       {"order":  0, "weight": 0.0, "fixed":   0, "copies": 2},
-        "named-item": {"order":  8, "weight": 0.0, "fixed":   0, "copies": 2}
-    }
+        "trial":      {"order":  1, "weight": 0.0, "fixed":   0, "copies": 1},
+        "always":     {"order":  2, "weight": 0.0, "fixed":   0, "copies": 1},
+        "woth":       {"order":  3, "weight": 0.0, "fixed":   5, "copies": 1},
+        "barren":     {"order":  4, "weight": 0.0, "fixed":   3, "copies": 1},
+        "entrance":   {"order":  5, "weight": 0.0, "fixed":   4, "copies": 1},
+        "sometimes":  {"order":  6, "weight": 0.0, "fixed": 100, "copies": 1},
+        "random":     {"order":  7, "weight": 9.0, "fixed":   0, "copies": 1},
+        "item":       {"order":  0, "weight": 0.0, "fixed":   0, "copies": 1},
+        "song":       {"order":  0, "weight": 0.0, "fixed":   0, "copies": 1},
+        "overworld":  {"order":  0, "weight": 0.0, "fixed":   0, "copies": 1},
+        "dungeon":    {"order":  0, "weight": 0.0, "fixed":   0, "copies": 1},
+        "junk":       {"order":  0, "weight": 0.0, "fixed":   0, "copies": 1},
+        "named-item": {"order":  8, "weight": 0.0, "fixed":   0, "copies": 1}
+    },
+    "groups": [
+        ["DMC (Bombable Wall)"],
```

Comment:
The singletons aren't necessary to list here?

Reply:
It's not, but I was explicitly listing every location because I assume this is going to change before it's PR'd. I don't know how people want to group them up, but I think the goal might be to drop to 14 groups, so there might not be any singletons.